lib/money/src/Converter.cpp

```diff
@@ -0,0 +1,31 @@
+#include "Converter.h"
+
+long Converter::ConvertToUSC(
+	double amount, Currency currency
+) const {
+	long USCAmount = { 0 };
```

Comment:
Don't use a temporary variable, in case statements use return.

Reply:
Please describe the problem in more detail.